lib/graphql/card.ts

```diff
@@ -283,4 +294,15 @@ export class Card extends Model<CardModel> {
     const result = await this.client.rawQuery(UPDATE_CARD_SETTINGS, args);
     return result.updateCardSettings;
   }
+
+  /**
+   * Replace a card
+   *
+   * @param args   query parameters including card id
+   * @returns      the newly created card details
+   */
+  async replaceCard(args: ReplaceCardOptions): Promise<CardModel> {
```

Comment:
what do you think of renaming this method just `replace`? otherwise usage would be `kontistClient.card.replaceCard` mentioning "card" twice is a bit redundant, also for other methods we took the same approach (e.g. `kontistClient.card.create`, etc.)

Reply:
Yeah, I has this idea after I already pushed it :) Especially since next method I need to implement is `reorderCard (reorder)`. It totally makes sense, will update it 👍